static/app/components/onboarding/gettingStartedDoc/utils/feedbackOnboarding.tsx

```diff
@@ -55,3 +58,23 @@ export function FeedbackOnboardingWebApiBanner() {
     </Alert>
   );
 }
+
+export const getFeedbackConfigOptions = ({
+  name,
+  email,
+}: {
+  email?: boolean;
+  name?: boolean;
+} = {}) => {
+  if (name && email) {
+    return `isNameRequired: true,
+          isEmailRequired: true,`;
+  }
+  if (name) {
+    return `isNameRequired: true,`;
+  }
+  if (email) {
+    return `isEmailRequired: true,`;
+  }
+  return ``;
+};
```

Comment:
Alternative proposal with less logical branches:
```suggestion
export const getFeedbackConfigOptions = ({
  name,
  email,
}: {
  email?: boolean;
  name?: boolean;
} = {}) => {
  const options: string[] = [];
  if (name) {
    options.push('isNameRequired: true,');
  }
  if (email) {
    options.push('isEmailRequired: true,');
  }
  return options.join('\n');
};
```